qe_metrics/cli.py

```diff
@@ -0,0 +1,71 @@
+import os
+from typing import Dict, Any
+
+import click
+
+from simple_logger.logger import get_logger
+
+from qe_metrics.libs.config import Config
+
+
+@click.group()
+@click.option(
+    "--config",
+    "-c",
+    default=os.environ.get("QE_METRICS_CONFIG") or "config.yaml",
+    help="Defines the path to the config file.",
+    type=click.Path(exists=True),
+)
+@click.option(
+    "--local-db",
+    is_flag=True,
+    help="Use a local SQLite database instead of a real database.",
+    type=click.BOOL,
+)
+@click.option(
+    "--pdb",
+    help="Drop to `ipdb` shell on exception",
+    is_flag=True,
+)
+@click.option(
+    "--verbose",
+    "-v",
+    is_flag=True,
+    help="Verbose output of database connection.",
+    type=click.BOOL,
+)
+def main(**kwargs: Dict[str, Any]) -> None:
+    """Gather QE Metrics"""
+
+    # Remove the pdb option from the kwargs
+    kwargs.pop("pdb", None)
+
+    # Adding noqa: F841 to ignore the unused variable until next PR, otherwise pre-commit will fail
+    config = Config(config_file=str(kwargs.get("config")))  # noqa: F841
+
+    # TODO: After getting database and Jira connection in config, use them to execute the queries here
+    # TODO: Populate the database with the results of the queries
+    # TODO: Run a cleanup of the database to remove old entries
+
+
+if __name__ == "__main__":
+    should_raise = False
+    _logger = get_logger(name="main-qe-metrics")
+    try:
+        main()
+    except Exception as ex:
+        import sys
+        import traceback
+
+        ipdb = __import__("ipdb")  # Bypass debug-statements pre-commit hook
```

Comment:
`ipdb` should move from `dev.dependencies` to `dependencies` section in pyproject.yaml

Reply:
Done, I also moved `ipython`. If you only think `ipdb` should be moved, let me know.

Reply:
only `ipdb` since it's part of the main code.

Reply:
Okay, moved ipython back to dev dependencies